features/cellular/framework/API/CellularDevice.h

```diff
@@ -314,6 +314,7 @@ class CellularDevice {
      */
     virtual CellularNetwork *open_network(FileHandle *fh = NULL) = 0;
 
+#if MBED_CONF_CELLULAR_USE_SMS || defined(DOXYGEN_ONLY)
     /** Create new CellularSMS interface.
      *
      *  @param fh    file handle used in communication to modem. This can be, for example, UART handle. If null, then the default
```

Comment:
Should the trick how to enable this via the `mbed_app.json`  target override be added here or?